documentation-website/cypress/e2e/results.cy.ts

```diff
@@ -11,7 +11,7 @@ describe('usage/results', () => {
     cy.url().should('eq', BASE_URL + page.results,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Results | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Results | Usage | @idrinth-api-bench/api-bench',);
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted